src/game_constants.h

```diff
@@ -71,7 +71,7 @@
 #define FRIDGE_TEMPERATURE 37 // 5 Celsius
```

Comment:
Is this correct?

![image](https://user-images.githubusercontent.com/16213433/44620576-82a64000-a89f-11e8-8483-a57fdfa2f62e.png)

Reply:
should be 41, thanks for catching that, I dunno what converter I used but it lied to me

Reply:
That said, I remember now, that I wanted them to be about half way between `FREEZING_TEMPERATURE` and `AVERAGE_ANNUAL_TEMPERATURE` to make them better then root cellars that use the latter temperature.
In that case its fine, just need to change the Celsius value in comment.